src/components/Tabs/Tabs.js

```diff
@@ -30,22 +31,54 @@ export default class Tabs extends React.Component {
   // Render current active tab content
   renderActiveTabContent() {
     const { children } = this.props;
-    const { activeTabIndex } = this.state;
+    const { activeTabIndex, horizontalLineWidth } = this.state;
     if (children[activeTabIndex]) {
-      return children[activeTabIndex].props.children;
+
+      let newChildren = [];
+      for (let i = 0; i < children[activeTabIndex].props.children.length; i++) {
+        let child = children[activeTabIndex].props.children[i];
+        console.log(child);
```

Comment:
If this is important enough, make this a 
`console.debug(log)`

Otherwise please take it out.

Reply:
Done :)